src/App.js

```diff
@@ -57,12 +58,19 @@ const App = () => {
                 </ProtectedRoute>
               }
             />
+            <Route 
+              path="cohortsStudent"
```

Comment:
This path should be `cohorts` - the actual view that the user sees should change depending on whether that user is a student or a teacher, they shouldn't be separate route paths

Reply:
This does not seem to be resolved. What Nathan meant is that the path should be cohorts (not chortsStudent)

Reply:
Yeah we unfortunately did not see Nathans comments till now! So we will do all the changes now and ask for a new review! Thats my bad!